pkg/deployment/config.go

```diff
@@ -34,8 +34,8 @@ const (
 	QueueSidecarImageKey = "queueSidecarImage"
 
 	// ProgressDeadlineDefault is the default value for the config's
-	// ProgressDeadlineSeconds.
-	ProgressDeadlineDefault = 120 * time.Second
+	// ProgressDeadlineSeconds. This matches the K8s default value of 600s.
+	ProgressDeadlineDefault = 600 * time.Second
```

Comment:
Let's discuss this change separately to avoid conflating a contentious thing with a non-contentious thing?

Reply:
Sure, let me revert back to 120.

Reply:
Done.